From 64d698c05198a5fb416e0ca66e3b813e7dfeccce Mon Sep 17 00:00:00 2001
From: Meng Li <meng.li@windriver.com>
Date: Mon, 12 Dec 2022 12:28:24 +0800
Subject: [PATCH 2/3] u-boot: secboot: correct the secure boot config

According to commit 4ace5bebb5f2("hse: rename secboot
options to generic hse "), it needs to use the new config
to enable secure boot feature.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-s32/s32-cc/Makefile | 2 +-
 doc/README.nxp-hse-secboot        | 4 ++--
 drivers/crypto/fsl/Makefile       | 2 +-
 drivers/crypto/fsl/hse/Makefile   | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/mach-s32/s32-cc/Makefile b/arch/arm/mach-s32/s32-cc/Makefile
index 0ee639ab..8d1d4e6d 100644
--- a/arch/arm/mach-s32/s32-cc/Makefile
+++ b/arch/arm/mach-s32/s32-cc/Makefile
@@ -9,7 +9,7 @@ obj-y += scmi_reset_agent.o
 obj-y += serdes_hwconfig.o
 obj-y += quick_boot_fixups.o
 obj-y += soc.o
-obj-$(CONFIG_HSE_SECBOOT)  += hse_adv_secboot.o
+obj-$(CONFIG_NXP_HSE_SUPPORT)  += hse_adv_secboot.o
 obj-y += start_m7.o
 obj-$(CONFIG_MP)		+= mp.o
 obj-$(CONFIG_OF_LIBFDT)	+= fdt.o
diff --git a/doc/README.nxp-hse-secboot b/doc/README.nxp-hse-secboot
index b44ad4ef..151a4376 100644
--- a/doc/README.nxp-hse-secboot
+++ b/doc/README.nxp-hse-secboot
@@ -22,8 +22,8 @@ Building U-Boot with HSE secure boot support
 The follow config options must be set:
 
     CONFIG_NXP_S32CC="y"
-    CONFIG_HSE_SECBOOT="y" (depends on CONFIG_NXP_S32CC)
-    CONFIG_CMD_HSE_ADVANCED_SECBOOT="y" (depends on CONFIG_HSE_SECBOOT)
+    CONFIG_NXP_HSE_SUPPORT="y" (depends on CONFIG_NXP_S32CC)
+    CONFIG_CMD_HSE_ADVANCED_SECBOOT="y" (depends on CONFIG_NXP_HSE_SUPPORT)
 
 Preparing U-Boot for secure booting
 -----------------------------------
diff --git a/drivers/crypto/fsl/Makefile b/drivers/crypto/fsl/Makefile
index 372c6691..d7ceb194 100644
--- a/drivers/crypto/fsl/Makefile
+++ b/drivers/crypto/fsl/Makefile
@@ -8,4 +8,4 @@ obj-$(CONFIG_CMD_BLOB)$(CONFIG_IMX_CAAM_DEK_ENCAP) += fsl_blob.o
 obj-$(CONFIG_RSA_FREESCALE_EXP) += fsl_rsa.o
 obj-$(CONFIG_FSL_CAAM_RNG) += rng.o
 obj-$(CONFIG_FSL_MFGPROT) += fsl_mfgprot.o
-obj-$(CONFIG_HSE_SECBOOT) += hse/
+obj-$(CONFIG_NXP_HSE_SUPPORT) += hse/
diff --git a/drivers/crypto/fsl/hse/Makefile b/drivers/crypto/fsl/hse/Makefile
index 175a526c..c2a84959 100644
--- a/drivers/crypto/fsl/hse/Makefile
+++ b/drivers/crypto/fsl/hse/Makefile
@@ -2,4 +2,4 @@
 #
 # Copyright 2020 NXP
 
-obj-$(CONFIG_HSE_SECBOOT) += hse_mu.o hse_mmc.o
+obj-$(CONFIG_NXP_HSE_SUPPORT) += hse_mu.o hse_mmc.o
-- 
2.34.1

